assets/out

```diff
@@ -63,7 +63,7 @@ for i in $(seq 0 $(expr "$files_count" - 1)); do
 done
 
 if [ ! -z "$build" ]; then
-  osc build $repository $arch
+  osc build --trust-all-projects $repository $arch
```

Comment:
You are right, an option would be the best, but with the current code base the `~/.oscrc` is overwritten in the Docker container by the resource. So there is no possibility at the moment to trust some projects via a specially created Docker container.

Of course we could remove the overwriting and only replace the username/password.